src/components/FilterPane/FilterPane.styles.js

```diff
@@ -86,23 +89,35 @@ export const StyledProjectListContainer = styled('div')`
   overflow-wrap: break-word;
 `
 
+export const StyledMethodListContainer = styled('div')`
+  margin-right: 1rem;
+  border: 1px solid ${theme.color.grey0};
+`
+
 export const StyledUnorderedList = styled('ul')`
   list-style-type: none;
-  padding: 0;
   margin: 0;
+  background-color: ${theme.color.white};
+  padding: 0;
+`
+
+export const StyledDateInputContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-right: 1rem;
+  justify-content: space-between;
 `
 
 export const StyledDateInput = styled.div`
   position: relative;
-  width: calc(50% - 0.3rem);
+  width: calc(50% - 0.5rem);
   margin-bottom: 1rem;
-  margin-right: 0.3rem;
   input {
     width: 100%;
     padding: 0.5rem;
     background-color: ${theme.color.white};
-    border: 1px solid #ccc;
```

Comment:
Should the `#ccc` colour be part of the theme?

Reply:
No the border shouldn't be there so it was removed and the #ccc too